x-pack/solutions/security/plugins/security_solution/common/entity_analytics/privileged_user_monitoring/constants.ts

```diff
@@ -7,7 +7,7 @@
 
 // Static index names: may be more obvious and easier to manage.
 export const PRIVMON_BASE_INDEX_NAME = '.entity_analytics.monitoring';
-export const ML_ANOMALIES_INDEX = '.ml-anomalies-shared';
+export const ML_ANOMALIES_INDEX = '.ml-anomalies-shared-*';
```

Comment:
To ensure full backwards compatibility, could we instead do:

```suggestion
export const ML_ANOMALIES_INDEX = '.ml-anomalies-shared*';
```